types/nightwatch/test/testElementCommands.ts

```diff
@@ -0,0 +1,505 @@
+import { NightwatchAPI, JSON_WEB_OBJECT, NightwatchSizeAndPosition } from "nightwatch";
+
+function isNull(v: null) {}
+function isNightwatchAPI(v: NightwatchAPI) {}
+function isStringOrNull(v: string | null) {}
+function isString(v: string) {}
+function isBoolean(v: boolean) {}
+function isElementSize(v: { width: number; height: number; x: number; y: number }) {}
+function isAny(v: any) {}
+function isJSON_WEB_OBJECT(v: JSON_WEB_OBJECT) {}
+function isNightwatchSizeAndPosition(v: NightwatchSizeAndPosition) {}
+function isObject(v: object) {}
+
+//
+// .clearValue
+//
+describe('clearValue Command demo', function() {
+    test('demo test', function() {
+        browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .clearValue('input[type=text]', function(result) {
+                isNightwatchAPI(this);
+            }).expect.element('input[type=text]').text.to.equal('');
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .clearValue('input[type=text]');
+        isNull(result);
+    });
+});
+
+//
+// .click
+//
+describe('click Command demo', function() {
+    test('demo test', function() {
+        browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .click('input[type=submit]', function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .click('input[type=submit]');
+        isNull(result);
+    });
+});
+
+//
+// .getAttribute
+//
+describe('getAttribute command demo', function() {
+    before(browser => browser.url('https://www.google.com/'));
+
+    test('demo test', function() {
+        browser
+            .getAttribute('input[type=text]', 'title' , function(result) {
+                isNightwatchAPI(this);
+            });
```

Comment:
Please check the returned type of `result` in all callback functions in this file:
```ts
function isNightwatchCallbackResult<T>(v: NightwatchCallbackResult<T>) {}
```
```suggestion
            .getAttribute('input[type=text]', 'title' , function(result) {
                isNightwatchAPI(this);
                isNightwatchCallbackResult<string | null>(result);
            });
```